.github/workflows/ci.yaml

```
@@ -107,11 +107,79 @@ env:
+
+            You may need to pull the latest changes before pushing again:
+            ```bash
+            git pull origin ${{ github.event.pull_request.head.ref }}
+            ```
+
+            **Triggered by commit:** `${{ github.event.pull_request.head.sha }}`
+            **Last formatted:** ${{ github.event.pull_request.updated_at }}
+            **Run**: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+
+      - name: git push
+        run: git push origin HEAD:${{ github.event.pull_request.head.ref }}
```

actionlint (1.7.8)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)